packages/hoppscotch-js-sandbox/src/utils.ts

```diff
@@ -86,9 +93,18 @@ const getSharedMethods = (envs: TestResult["envs"]) => {
 
     const result = pipe(
       getEnv(key, updatedEnvs),
-      O.match(
+      O.fold(
         () => undefined,
-        ({ value }) => String(value)
+        (env) =>
+          String(
+            (
+              env as {
+                key: string
+                value: string | undefined
+                secret: boolean
+              }
```

Comment:
Since we know from the sandbox, that environment variables will follow the above shape, can we maintain a custom type under `src/types` or similar that can be reused across?